BoundingBoxes bounding_boxes
int32[] track_ids
